### PR TITLE
QA Process - Documentation

### PR DESCRIPTION
**Pull request by robindevitt**
This pr adds a Codeception test that checks the documentation renders correctly ( Acceptance Test ).

[Original Issue ](https://github.com/humanmade/product-dev/issues/837)

### CODE REVIEW

**Comment by robindevitt** [comment]
Hey @shadyvb if you get a gap to review this PR for me that would be great 👍🏼 

HM Linter is failing due to a minified CSS file, nothing related to this issue.

**Review by shadyvb — CHANGES_REQUESTED** [review]
@robindevitt can we also make sure the menu exists and has items, and that the content is populated ? just in case the content rendering hits a bug or something.

**Comment by robindevitt** [comment]
> @robindevitt can we also make sure the menu exists and has items, and that the content is populated ? just in case the content rendering hits a bug or something.

@shadyvb I have add a check that makes sure the documentation link is shown as well as a test that navigates to the CMS module and in the nav and then confirms the CMS H1 is seen.

This should cover all the bases?

File: tests/acceptance/DocumentationCest.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		$I->loginAsAdmin();
+		$I->amOnAdminPage( 'admin.php?page=altis-documentation' );
+
+		// CMS Module.
+		$I->see( 'CMS', 'li' );
+		$I->click( 'CMS' );
+
+		// See the CMS H! title.
+		$I->see( 'CMS', 'h1' );
+	}
```
Comment (shadyvb) [inline tests/acceptance/DocumentationCest.php:61]: I don't think it is necessary to do three separate tests to check the same page, it adds 3 times the requests. One test should be fine. eg:
```suggestion
	/**
	 * Documantation link is shown, and page renders correctly.
	 *
	 * @param AcceptanceTester $I Tester
	 */
	public function testDocumentationLink( AcceptanceTester $I ) {
		$I->wantToTest( 'Documantation link is shown, and page renders correctly.' );
		$I->loginAsAdmin();
		$I->amOnAdminPage( '/' );

		// See the Documentation link in menu.
		$I->moveMouseOver( '.altis-logo-wrapper' );
		$I->seeLink( 'Documentation' );
		
		// Click the link to open the documentation.
		$I->click( 'Documentation' );
		
		// See the main title.
		$I->seeElement( '.altis-ui__doc-title' );
		
		// Click to go to CMS Module docs.
		$I->see( 'CMS', 'li' );
		$I->click( 'CMS' );

		// See the CMS H1 title.
		$I->see( 'CMS', 'h1' );
	}
}
```

**Review by shadyvb — CHANGES_REQUESTED** [review]
LGTM, just a small suggestion to improve the running time by removing 4 additional requests.

**Review by shadyvb — APPROVED** [review]

**Pull request merged**